web/styles/settings.css

```diff
@@ -247,18 +247,18 @@ h3,
     }
 
     .loading_indicator_text {
-        font-size: 12px;
+        font-size: 0.8571em; /* 12px at 14px/em */
         font-weight: 400;
         vertical-align: middle;
-        line-height: 20px;
+        line-height: 1.6667em; /* 20px at 12px/em */
```

Comment:
Why is this calculated as per 12px font size and not 14px? Vertical alignment looks off at 12px to me.
![Screenshot from 2025-03-06 19-16-51](https://github.com/user-attachments/assets/6b5d87a7-d1ef-4ae2-92b6-ff99d2cdb1af)

Reply:
Because the font-size for this element is 12px, so the 20px from before was at 12px.